internal/test/framework/windowsvm.go

```diff
@@ -64,6 +66,12 @@ type WindowsVM interface {
 	Reinitialize() error
 	// Destroy destroys the Windows VM
 	Destroy() error
+	// GetBuildWmcb gets the value of buildWmcb. If value is true, WSU will build WMCB before transferring. If value is
+	// false, WSU will download the latest released version of WMCB based on the cluster version
+	GetBuildWmcb() bool
```

Comment:
I agree with Sebastain. @ravisantoshgudimetla and I were discussing moving `WindowsVM` to WNI so that we can reuse it in WMCO. Once we do that, we will remove this from the framework. Then in the framework it can be specialized as `TestWindowsVM`.

Reply:
Or compose your interface with other interface called TestWindowsVM and have these fields there. In either case, that's a good call.